public/Day 134/index.html

```
@@ -0,0 +1,513 @@
+                    <div class="flex items-center justify-between text-indigo-600">
+                        <div class="flex items-center gap-3">
+                            <i data-lucide="layout" class="w-5 h-5"></i>
+                            <h2 class="text-xs font-black uppercase tracking-[0.2em]">Composer</h2>
+                        </div>
+                        <button onclick="resetInputs()" class="text-[10px] font-bold text-slate-400 hover:text-red-500 transition-colors">RESET</button>
+                    </div>
+
+                    <div class="space-y-5">
+                        <div class="space-y-1.5">
+                            <label class="text-xs font-bold text-slate-400 ml-1">Recipient Name</label>
+                            <input id="recipient" type="text" placeholder="e.g. Sarah Jenkins" class="w-full px-5 py-3.5 bg-slate-50 border border-slate-200 rounded-2xl focus:ring-2 focus:ring-indigo-500 focus:bg-white outline-none transition-all text-sm font-medium">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ input ] must be self closed.

(tag-self-close)